Introduce hello world in Ruby

githubteacher opened the pull request:
@fooforge What do you think of this change?

githubteacher committed:
Say hello to the world from Ruby

githubteacher committed:
Use proper casing for world

fooforge commented:
LGTM, but needs more BATMAN! :metal:

```
       _==/          i     i          \==_
     /XX/            |\___/|            \XX\
   /XXXX\            |XXXXX|            /XXXX\
  |XXXXXX\_         _XXXXXXX_         _/XXXXXX|
 XXXXXXXXXXXxxxxxxxXXXXXXXXXXXxxxxxxxXXXXXXXXXXX
|XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX|
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
|XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX|
 XXXXXX/^^^^"\XXXXXXXXXXXXXXXXXXXXX/^^^^^\XXXXXX
  |XXX|       \XXX/^^\XXXXX/^^\XXX/       |XXX|
    \XX\       \X/    \XXX/    \X/       /XX/
       "\       "      \X/      "      /"
```

githubteacher commented:
@fooforge Totally agree with you. I will add more BATMAN! :sparkling_heart:

githubteacher committed:
Adding more BATMAN

@fooforge had the excellent suggestion we need more BATMAN. This introduces it and replaces the hello world message with a Batman ASCII art.

fooforge commented:
Ship it! :+1: :shipit:
